RevenueCat/Editor/RevenueCatPostInstall.cs

```diff
@@ -1,5 +1,5 @@
 
-#if UNITY_EDITOR && UNITY_IOS
+#if UNITY_EDITOR && (UNITY_IOS || UNITY_VISIONOS)
```

Comment:
I haven't been able to test this, but do we need to also check for vision os in line 16 below? https://docs.unity3d.com/ScriptReference/BuildTarget.VisionOS.html

Reply:
Does it support Unity-VisionOS now?

Reply:
After testing with the new `BuildTarget.VisionOS`, this actually brakes with the current unity version we are using and it would be considered a breaking change. In any case, this script shouldn't really be needed in modern unity versions so we can merge without it.